test/test_mdlp.py

```diff
@@ -19,21 +11,21 @@ def test_slice_entropy():
     entropy1, k1 = slice_entropy(y, 0, 3)
     entropy2, k2 = slice_entropy(y, 3, 10)
 
-    assert_equal(entropy1, 0, "Entropy was not calculated correctly.")
-    assert_equal(k1, 1, "Incorrect number of classes found.")
-    assert_almost_equal(entropy2, 0.796311640173813,
-                        err_msg="Entropy was not calculated correctly.")
-    assert_equal(k2, 3, "Incorrect number of classes found.")
+    assert k1 == 1
```

Comment:
What's wrong with `assert_equal`? 

I'd prefer `assert_equal` here, for consistency.

Reply:
sounds good.